engine/previews/citizens_advice_components/sidebar_preview/with_additional_content.html.haml

```diff
@@ -0,0 +1,12 @@
+= render CitizensAdviceComponents::Sidebar.new(title: "Related Content") do |c|
+  - c.section_title(section_title)
+  - c.sidebar_links(sidebar_links)
+  .example-links-block= link_to "Adviser resources - EU Settlement Scheme", "/advisernet/immigration/staying-in-the-uk/settled-status/adviser-resources-eu-settlement-scheme"
```

Comment:
Not sure if this should have a special class like `cads-sidebar__additional-content`? I suppose it can just use the stying of the additional content used (e.g advisor link block)

Reply:
Yeah I think people can style whatever they want to put in, so I don't think we need it.  If they have a specificity issue they can always use `.cads-sidebar` instead.